packages/visualizations/addon/utils/chart-data.js

```diff
@@ -97,7 +97,7 @@ export function chartTypeForRequest(request) {
  * @returns {Array} - list of metric names
```

Comment:
Looks like this return statement is invalid now.
And this method doesn't sound like it belongs here.